src/Quotations.ts

```diff
@@ -167,7 +168,8 @@ function extractQuotationUsingCheckpoints(xmlDocument: Document, messageBody: st
   // Serialize and return.
   return {
     body: xmlDomSerializer.serializeToString(xmlDocument, true),
-    didFindQuote: true
+    didFindQuote: true,
+    usingCheckpoints: true
```

Comment:
Should we call that `didUseCheckpoints` for consistency with the other booleans?

Reply:
good point 👍